plotly/session.py

```diff
@@ -39,7 +39,8 @@
     'fileopt': six.string_types,
     'world_readable': bool,
     'auto_open': bool,
-    'validate': bool
+    'validate': bool,
+    'sharing': dict
```

Comment:
:cow2: ha, back to where we were before, we don't have enough information to validate this. I think it's OK as is though. In fact, I'm convinced the *right* way to do this is validation via JSON schemas http://json-schema.org/

I'll open up a ticket for that too. :)

If you *wanted*, you could add a check to ensure that `sharing in ('public', 'private', 'secret')`. It's really good to lock down the JSON, no surprises!